bambu_spoolman/broker/filament_usage_tracker.py

```diff
@@ -121,8 +121,12 @@ def _retrieve_model(self, model_url):
 
         if uri.scheme == "https" or uri.scheme == "http":
             return self._download_model(model_url)
-        elif uri.scheme == "file":
-            return self._retrieve_model_from_ftp(uri.path)
+        elif uri.scheme == "file" or uri.scheme == "ftp":
+            if uri.path:
```

Comment:
Admittedly, I somewhat blindly wrote this code, as I send my prints through Bambu's cloud, so its very likely this has always been somewhat broken.

We should probably construct the path in a more correct way. Something like
```py
path = f"{uri.netloc}{uri.path}"
```
This should allow the correct path to be constructed regardless of location on the SD card.